import_export/templates/admin/import_export/export.html

```diff
@@ -16,6 +16,22 @@
 <form action="" method="POST">
   {% csrf_token %}
 
+  {% block fields_help %}
+    <p>
+      {% trans "This exporter will export the following fields: " %}
+      {% if fields_list|length <= 1 %}
```

Comment:
Lines 22-31 are exactly the same as those in import.html - could they be added to a [shared template file](https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#include)?

Reply:
done